convert/lib/convert.py

```diff
@@ -443,12 +454,25 @@ def get_subnet_ids(subnet_type: str, prefix: str = "VPC"):
             "eks_cluster_auto_sg": eks_cluster_auto_sg,
         }
 
+        sse_kms_key_ids = [v["sse_kms_key_id"] for k, v in self.cdkconfig["s3"]["buckets"].items()]
+        sse_kms_key_ids.append(self.cdkconfig["eks"]["secrets_encryption_key_arn"])
+        kms_id_note = False
+        if not all(True for key_id in sse_kms_key_ids if key_id is None):
```

Comment:
just a suggestion, it's clearer in my head

```suggestion
        if any(key_id is not None for key_id in sse_kms_key_ids):
```